psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -436,6 +442,7 @@ def scf_iterate(self, e_conv=None, d_conv=None):
         core.timer_off("HF: Form D")
 
         self.set_variable("SCF ITERATION ENERGY", SCFE)
+        core.set_variable("SCF D NORM", Dnorm)
```

Comment:
It's interesting to me that the "SCF D NORM" variable is part of core rather than self. Is there any reason for this, and would it be better for SCF D NORM to be part of self here instead of core?

Reply:
Good catch -- it alarmed me, too, at one point. iirc, the answer was that directJK that reads this doesn't have access to the wfn object, so globals is the easiest way to communicate.

Reply:
Ahhhhhhhh, I see. That would make sense!